src/libs/decoders/stb_vorbis.h

```diff
@@ -3834,8 +3913,8 @@ static int start_decoder(vorb *f)
                unsigned int div=1;
                for (k=0; k < c->dimensions; ++k) {
                   int off = (z / div) % c->lookup_values;
-                  float val = mults[off];
-                  val = mults[off]*c->delta_value + c->minimum_value + last;
+                  // float val = mults[off]; // under review at https://github.com/nothings/stb/issues/816
```

Comment:
Thanks for this link… I am a bit worried by the number of open issues and unmerged pull requests in STB's repo :(

Reply:
Although it looks unkept on the surface, his repo gets a ton of traffic and he does make his way through the reports, albeit slower than Dave of dr_libs.

For us though  stb_vorbis is an early creation from 12 years ago (pre-Opus) and was funded by RAD game tools (now used in over 23,000 games!).  So the library is extremely battle-hardened and functional fixes are quite rare now.  Sean does address them; two versions ago I report the skeleton ogg issue (which became v1.15 of stb_vorbis) and a patch, which he accepted:

https://github.com/nothings/stb/commit/604b9367ee7a77aeff916a6bb5a9ee9601d72c02

Dr_libs are much younger and I suspect used more rarely (which is unfortunate, because they are of impeccable quality and performance) For example, dr_mp3 hadn't been tested (and didn't work) on my big-endian PPC7400, and Dave was very interested in getting that working. Same with the fast seeking; that was some awesome he did and we collaborated last winter.

I'm rambling now, but yes, when Sean leaves issues open, it's my interpretation that those are the ones he plans to address in the next revision cycle. Might be another 6 months or so before that happen with stb_vorbis though!

Reply:
Another very exciting this about stb's libraries is that they have been selected by Google's fuzzing team.  I would love to see all the codecs fuzzed. So far Sean hasn't responded Google's offer, but he also hasn't closed it, whcih tells me he's probably interested (maybe if someone submit a patch). 
This criteria is pretty significant too:
"In order for a project to be accepted to OSS-Fuzz, it needs to have a large user base and/or be critical to Global IT infrastructure"